src/unix/open.c

```diff
@@ -365,7 +365,7 @@ value caml_gr_sigio_handler(void)
 
 /* Processing of graphic errors */
 
-static value * graphic_failure_exn = NULL;
+const static value * graphic_failure_exn = NULL;
```

Comment:
I'm not an expert in the C `const` keyword, but intuitively I would interpret this as "graphic_failure_exn is a constant", which it isn't. Shouldn't we cast the result of `caml_named_value` instead?

Reply:
> I would interpret this as "graphic_failure_exn is a constant"

No, that would be `static value * const graphic_failure_exn`.

`const static value * graphic_failure_exn` means that `graphic_failure_exn` is a pointer to a value that should not be modified (through this pointer at least).

That said I would prefer `static const value * graphic_failure_exn`.

Reply:
Ok, thanks for the explanation.

Regarding `const static` vs `static const`, after doing a bit of googling about it I agree that `static const` seems better.

@madroach could you swap `const` and `static`? I'll merge after that.

Thanks